src/Sulu/Bundle/PageBundle/Markup/Link/PageLinkProvider.php

```diff
@@ -103,8 +103,17 @@ public function preload(array $hrefs, $locale, $published = true)
             $domain = $request->getHost();
         }
 
+        $uuids = \array_filter(
```

Comment:
I would handle this things outside of the specific link providers. So the `PageLinkProvider` does only get the `uuids` without the `#` and adding the `#hash` at the end is in a more general way say it will in future work for all LinkProviders without the LinkProvider need to take care of the `#` anchors.

Reply:
Think the `LinkTag` service would I think the best place for it.